retrieval/raw/raw_endpoint.go

```diff
@@ -43,7 +48,25 @@ func (c *Client) createRequest(method, url string, body io.Reader) (*http.Reques
 }
 
 //GetRawEndPoint retrives the body of HTTP response from a given sourcename, working directory, and URL
-func (c *Client) GetRawEndPoint(sourceName string, workDir *os.File, URL string) (rawRespFile *os.File, rerr error) {
+func (c *Client) GetRawEndPoint(sourceName string,
+	workDir *os.File, URL string) (rawRespFile *os.File, err error) {
+
+	attempts := c.retries + 1
+
+	for i := uint(0); i < attempts; i++ {
+		rawRespFile, err := downloadToFile(c, sourceName, workDir, URL, i)
```

Comment:
we should just use `=` here vs `:=` otherwise it shadows the return variables in this for loop